Add dictionary definition for "JavaScript" (#193)
cherryontech/website: src/assets/content/dictionary/javascript.md

---
title: JavaScript 
definition: 'JavaScript is a programming or scripting language that allows one to create a dynamic or complex webpage.
 Features such as interactive maps, animated graphics, multimedia functionalities and many more are all created with Javascript. It is referred to as the third layer in the three layers of standard web technologies'
sources:
- sourceurl: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript
perspectives:

> - meaning: is a programming language used to bring concepts to life. It is responsible for implementing designs and site interactions

Reviewer: ```suggestion
- meaning: a programming language I can use to bring concepts to life. I can use it to implement designs and site interactions.
```
